.github/workflows/release.yaml

```diff
@@ -0,0 +1,33 @@
+name: cprover-release
+on:
+  # Only on pull request so I can see what's happening
+  pull_request:
+    branches: [develop]
+  release:
+    types: [created]
```

Comment:
This is currently triggered "on release", but more likely it would be better to have an automatically scheduled released (via the schedule event + release action).